apps/ejabberd/src/mod_http_notification.erl

```diff
@@ -115,3 +89,14 @@ record_result(Host, {error, Reason}, _) ->
     mongoose_metrics:update([Host, mod_http_notifications, failed], 1),
     ?WARNING_MSG("Sending http notification failed: ~p", [Reason]),
     ok.
+
+%% @doc Convert to binary, strip initial slash (it is added by mongoose_http_client)
+fix_path(P) when is_list(P) ->
```

Comment:
Since we should expect ordinary string from the config file, maybe it would be more readable to have only one variant of the `fix_path` function which accepts binary only. The conversion from list to binary could be done where the function is called. What do you think?